setup.py

```diff
@@ -131,8 +130,9 @@ def combine_files(*args):
     scripts=[
         'bin/tower-cli',
     ],
-    packages=find_packages(),
-
+    packages=[
+        'tower_cli',
+    ],
```

Comment:
And of course 'find_packages(exclude=['tests'])' may work but I just thought and felt that it might be a little excessive as only tower_cli to install as a module.